### ISSUE TITLE
 Range violation

### ISSUE THREAD

**Issue by o3o** [AuburnSounds/printed#36]
```
$ cd printed/
$ git describe --tags                                                                                                                                        128 (0.014s) < v1.0.5

$ cd example/simple
$ dub run -q                                                                                                                                           0 (0.001s) < 08:32:51
core.exception.RangeError@../../font/printed/font/opentype.d(394): Range violation
----------------
??:? _d_arrayboundsp [0x5651707c0095]
../../font/printed/font/opentype.d:394 int printed.font.opentype.OpenTypeFont.horizontalAdvance(dchar) [0x56517079f0e7]
../../font/printed/font/opentype.d:442 int printed.font.opentype.OpenTypeFont.measureText(immutable(char)[]).__foreachbody4(ref dchar) [0x56517079f2d3]
??:? _aApplycd1 [0x5651707c07e1]
../../font/printed/font/opentype.d:441 printed.font.opentype.OpenTypeTextMetrics printed.font.opentype.OpenTypeFont.measureText(immutable(char)[]) [0x56517079f292]
../../canvas/printed/canvas/pdfrender.d:158 void printed.canvas.pdfrender.PDFDocument.fillText(immutable(char)[], float, float) [0x565170769c9d]
source/app.d:69 _Dmain [0x565170756ae6]
Program exited with code 1
```
Maybe depends on #25?

Thank you

**Comment by p0nce** [comment]
Reproduced using default font, resolution TBD.

**Comment by p0nce** [comment]
Your problem should be fixed in printed v1.0.6
